zipline/test_algorithms.py

```diff
@@ -703,8 +725,8 @@ def handle_data(self, data):
                 self.return_nan.handle_data(data))
         else:
             nan_data = deepcopy(data)
-            for sid in nan_data.iterkeys():
-                nan_data[sid].price = np.nan
+            for sidint in nan_data.iterkeys():
```

Comment:
Is this just setting a column on a DataFrame to np.nan?  If so, it should be written as `nan_data.price = np.nan`.